config/300-eventing.yaml

```diff
@@ -941,7 +941,7 @@ spec:
       type: string
   - name: v1beta1
     served: true
-    storage: false
+    storage: true
```

Comment:
Do we run any storage version migrator for this? 

/cc @houshengbo @lionelvillard